jetson/odrive_bridge/src/__main__.py

```diff
@@ -50,23 +50,30 @@ def main():
     # starting state is DisconnectedState()
     # start up sequence is called, disconnected-->disarm-->arm
 
+    prev_comms = False
+
     while True:
         watchdog = t.clock() - start_time
         if (watchdog > 1.0):

```

Comment:
Did you lint this? I think Pylint complains about having parentheses in conditional statements

Reply:
passes the linter & travis so yes